src/components/routes/PrivateRoute.js

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import { bool, string, node } from 'prop-types';
-import { Route, Redirect, useLocation } from 'react-router-dom';
+import { Route, Redirect, useLocation, Link } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
 
 import routes from 'constants/routesPaths';
 import { useSession } from 'hooks';
 
-const PrivateRoute = ({ children, exact = false, path }) => {
+const PrivateRoute = ({ children, exact = false, path, hideMenu }) => {
```

Comment:
Maybe add a default value to `hideMenu`

Reply:
It should evaluate to false if nothing is passed, but to be concise, I see the exact prop has the default value, I'll apply the change